externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngineListener.scala

```diff
@@ -17,13 +17,16 @@
 
 package org.apache.kyuubi.engine.spark
 
-import org.apache.spark.scheduler.{SparkListener, SparkListenerApplicationEnd}
+import org.apache.spark.scheduler.{JobFailed, SparkListener, SparkListenerApplicationEnd, SparkListenerJobEnd}
 
 import org.apache.kyuubi.Logging
+import org.apache.kyuubi.config.KyuubiConf.ENGINE_DEREGISTER_EXCEPTION_CLASSES
 import org.apache.kyuubi.service.{Serverable, ServiceState}
 
 class SparkSQLEngineListener(server: Serverable) extends SparkListener with Logging {
 
+  val deRegisterExceptionCLs = server.getConf.get(ENGINE_DEREGISTER_EXCEPTION_CLASSES)
```

Comment:
`CLs` or `Cls`?

Reply:
yeah, I wonder that Cls means a singleton class, So, I try to use CLs.

Reply:
deregisterExceptions?

Reply:
but there are exception classes, exception messages ...

Reply:
deregisterMessages deregisterStacktraces